os/fs/driver/mtd/smart.c

```diff
@@ -124,8 +133,9 @@
 #define SMART_FMT_SIG4            'T'
 
 #define SMART_FMT_VERSION_POS     (SMART_FMT_POS1 + 4)
-#define SMART_FMT_NAMESIZE_POS    (SMART_FMT_POS1 + 5)
-#define SMART_FMT_ROOTDIRS_POS    (SMART_FMT_POS1 + 6)
+#define SMART_FMT_JOURNAL_POS     (SMART_FMT_POS1 + 5)
```

Comment:
Any particular reason for adding this before NAMESIZE and ROOTDIR signature bytes?
Order of significance?

Reply:
Journaling is tightly coupled with crc, so I added here.

Reply:
Fair enough